networkmanager/networkmanager_test.go

```diff
@@ -18,37 +18,80 @@
 package networkmanager_test
 
```

Comment:
Header in commit message should be 75 symbols len. See https://kb.epam.com/display/EPMDAEPRA/Coding+style+and+commits+submission

Reply:
Done